docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -0,0 +1,840 @@
+Multi<String> multiFromMaybe = Multi.createFrom().converter(MultiRxConverters.fromMaybe(), maybe);
+Multi<String> multiFromEmptyMaybe = Multi.createFrom().converter(MultiRxConverters.fromMaybe(), emptyMaybe);
+Multi<String> multiFromObservable = Multi.createFrom().converter(MultiRxConverters.fromObservable(), observable);
+Multi<String> multiFromFlowable = Multi.createFrom().converter(MultiRxConverters.fromFlowable(), flowable);
+----
+
+You can also transform `Unis` and `Multis` into RxJava types:
+
+[source, java]
+----
+Completable completable = uni.convert().with(UniRxConverters.toCompletable());
+Single<Optional<String>> single = uni.convert().with(UniRxConverters.toSingle());
+Single<String> single2 = uni.convert().with(UniRxConverters.toSingle().failOnNull());
+Maybe<String> maybe = uni.convert().with(UniRxConverters.toMaybe());
+Observable<String> observable = uni.convert().with(UniRxConverters.toObservable());
+Flowable<String> flowable = uni.convert().with(UniRxConverters.toFlowable());
+// ...
+Completable completable = multi.convert().with(MultiRxConverters.toCompletable());
+Single<Optional<String>> single = multi.convert().with(MultiRxConverters.toSingle());
+Single<String> single2 = multi.convert().with(MultiRxConverters
+        .toSingle().onEmptyThrow(() -> new Exception("D'oh!")));
+Maybe<String> maybe = multi.convert().with(MultiRxConverters.toMaybe());
+Observable<String> observable = multi.convert().with(MultiRxConverters.toObservable());
+Flowable<String> flowable = multi.convert().with(MultiRxConverters.toFlowable());
+----
+
+Project Reactor converters are available in the following dependency:
+
+[source, xml]
+----
+<dependency>
+    <groupId>io.smallrye.reactive</groupId>
+    <artifactId>mutiny-reactor</artifactId>
+    <version>$VERSION</version>
+</dependency>
+----
+
+So if you have an API returning Reactor types (`Mono`, `Flux`), you can create `Unis` and `Multis` as follows:
+
+[source, java]
+----
+import io.smallrye.mutiny.converters.multi.MultiReactorConverters;
+import io.smallrye.mutiny.converters.uni.UniReactorConverters;
+// ...
+Uni<String> uniFromMono = Uni.createFrom().converter(UniReactorConverters.fromMono(), mono);
+Uni<String> uniFromFlux = Uni.createFrom().converter(UniReactorConverters.fromFlux(), flux);
+
+Multi<String> multiFromMono = Multi.createFrom().converter(MultiReactorConverters.fromMono(), mono);
+Multi<String> multiFromFlux = Multi.createFrom().converter(MultiReactorConverters.fromFlux(), flux);
+----
+
+You can also transform `Unis` and `Multis` into Reactor types:
+
+[source, java]
+----
+Mono<String> mono = uni.convert().with(UniReactorConverters.toMono());
+Flux<String> flux = uni.convert().with(UniReactorConverters.toFlux());
+
+Mono<String> mono2 = multi.convert().with(MultiReactorConverters.toMono());
+Flux<String> flux2 = multi.convert().with(MultiReactorConverters.toFlux());
+----
+
+=== Using API using CompletionStages or Publisher
```

Comment:
Maybe rename the title as 
```
=== Using CompletionStages or Publisher API
```
Th current one feels strange to me.

Reply:
Fixed.